src/harness/values.yaml

```diff
@@ -109,70 +120,85 @@ global:
       hosts:
         - "myhostname.example.com"
 
+# -- Install the Continuous Integration (CI) manager pod
 ci:
   ci-manager:
     affinity: {}
     nodeSelector: {}
     tolerations: []
 
+# -- Config for Security Test Orchestration (STO) 
 sto:
+  # -- Install the STO core
```

Comment:
These are not installing the STO core.
We can call them: "Overrides for STO-core"